apps/system/js/system_dialog_manager.js

```diff
@@ -55,9 +55,12 @@
      * @memberof SystemDialogManager#
      */
     configs: {
-      listens: ['system-dialog-created',
-                'simlockcreated',
+      listens: ['system-dialog-started',
+                'system-dialog-stopped',
+                'simlockstarted',
+                'simlockstopped',
                 'actionmenucreated',
+                'actionmenudestroyed',
```

Comment:
When does `actionmenudestroyed` get called. I only see activities.js and multi_screen_contoller.js use it, and they don't declare it in their dependency lists but rather instantiate it directly.